tests/h/util/test_logging_filters.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+"""Test logging filters."""
+from __future__ import unicode_literals
+
+import pytest
+import logging
+from requests.exceptions import ReadTimeout
+from h.util.logging_filters import FilterExceptions
+
+
+class TestLoggingFilters(object):
+    """Test logging filters."""
+
+    def test_filter_bad_level(self):
+        with pytest.raises(ValueError):
+            FilterExceptions((("requests.exceptions.ReadTimeout", "WARNI"),))
+
+    def test_filter_wrong_exception_name(self):
+        with pytest.raises(ValueError):
+            FilterExceptions((("requests.exceptions.Bugga", "WARN"),))
+
+    def test_filter_wrong_module_name(self):
+        with pytest.raises(ValueError):
+            FilterExceptions((("requests.bad.ReadTimeout", "WARN"),))
+
+    def test_filter_level_as_num(self, logger):
+        FilterExceptions((("requests.exceptions.ReadTimeout", logging.WARNING),))
+
+    def test_filter_no_log(self, logger):
```

Comment:
A general note about test naming - we usually use BDD-style naming for tests to make it easier to see at a glance what the test checks for. eg. `test_filter_removes_matching_exceptions` or `test_raises_if_wrong_module_name`. It took me a couple of reads through to figure out which were the success vs. error handling cases.